public/app/plugins/panel/timeseries/config.ts

```diff
@@ -42,7 +42,9 @@ export const defaultGraphConfig: GraphFieldConfig = {
 
 const categoryStyles = ['Graph styles'];
 
-export function getGraphFieldConfig(cfg: GraphFieldConfig): SetFieldConfigOptionsArgs<GraphFieldConfig> {
+type NullEditorSettings = { isTime: boolean };
+
+export function getGraphFieldConfig(cfg: GraphFieldConfig, isTime = true): SetFieldConfigOptionsArgs<GraphFieldConfig> {
```

Comment:
are we happy with `isTime` naming? Maybe it would be better as `isXAxisTime` or something more explicit?

Reply:
i'm okay with `isTime` since the editor itself is generic and this flag just indicates if we want time parsing for the input.

that we use it for x axis in some specific case shouldn't affect the opt naming here.